examples/view_multiglobe.html

```diff
@@ -110,9 +110,14 @@
                 if (evt.keyCode == 32) {
                     front = 1 - front;
                     t = 0;
+
                     // schedule the animation
                     view.addFrameRequester(itowns.MAIN_LOOP_EVENTS.BEFORE_RENDER, animator);
                     view.notifyChange(cameraThree);
+
+                    view.camera3D.near = 15;
+                    view.camera3D.far = 63781370;
+                    view.camera3D.updateProjectionMatrix();
```

Comment:
This fixes a functional test.
The near/far dynamic calculation assumes that the globe is centered at the origin. If the user wants to hack that position, maybe they are supposed to hack (i.e. overwrite) the near/far as well?